airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -382,12 +383,11 @@ def get_pod_log_stream(
         :param container: container name
         :param namespace: kubernetes namespace
         """
-        api = client.CoreV1Api(self.api_client)
         watcher = watch.Watch()
         return (
             watcher,
             watcher.stream(
-                api.read_namespaced_pod_log,
+                self.core_v1_client.read_namespaced_pod_log,
```

Comment:
This is the refactor, and 1 more identical below it. The rest are typing changes.